### ISSUE TITLE
Feature request: support for brotli

### ISSUE THREAD

**Issue by M-Gonzalo** [Gusb3ll/elysia-compression#3]
Any plans to add brotli to the mix? It's the most efficient code to date, and widely supported in all modern clients.
Thanks in advance!

**Comment by Gusb3ll** [comment]
Seem like `brotliCompressSync` hasn't been implemented on bun yet so I'm looking for a workaround rn

```
SyntaxError: Import named 'brotliCompressSync' not found in module 'node:zlib'
```

**Comment by M-Gonzalo** [comment]
Yes, it might take using an external binary but iDK if bun supports that type of module installation. I never could install anything that needs a compiling phase.
Thanks for looking into it though. Let me know if I can help in anything towards this goal.

**Comment by Gusb3ll** [comment]
> Yes, it might take using an external binary but iDK if bun supports that type of module installation. I never could install anything that needs a compiling phase. Thanks for looking into it though. Let me know if I can help in anything towards this goal.

Aight, I've already created a feature request on bun rn, https://github.com/oven-sh/bun/issues/6152
